### ISSUE TITLE
A question about SYNPROXY with Dynamic IP.

### ISSUE THREAD

**Issue by eromoe** [firehol/firehol#247]
Hi,

From https://github.com/firehol/firehol/wiki/Working-with-SYNPROXY-and-traps, I don't understand this 
> We added src not "${UNROUTABLE_IPS}" dst not "${UNROUTABLE_IPS}" to synproxy since this what we know (we don't want traffic generated by our network to be matched, and we don't know our public IP, but we know it is an UNROUTABLE one.

What is `UNROUTABLE`,  I am in Canada and have a openvz vps in USA. Is this `UNROUTABLE` or not ?

**Comment by philwhineray** [comment]
I think that should probably read `but we know it is not an UNROUTABLE one`, actually, given the `not`s and the fact it is public. @ktsaou ?

Unroutable addresses are private ones, e.g. `10.0.0.0/8` and reserved ones e.g. `127.0.0.0/8`. See `RESERVED_IPV4`, `RESERVED_IPV6`, `PRIVATE_IPV4` and `PRIVATE_IPV6` in `/etc/firehol/firehol.conf`.

**Comment by ktsaou** [comment]
Exactly. `${UNROUTABLE_IPS}` includes all the IPs that are routed in the internet.

**Comment by ktsaou** [comment]
not routed - I fixed it.
